cmd/search/static-services-entries.json

```diff
@@ -144,28 +144,28 @@
       "id": "OPENSHIFT.cluster.create.aws",
       "href": "/openshift/create",
       "title": "Red Hat OpenShift Service on AWS",
-      "alt_title": ["ROSA", "AWS", "OpenShift on AWS"] 
+      "alt_title": ["ROSA", "AWS", "OpenShift on AWS"]
```

Comment:
These custom links will require a description as well. Otherwise, the UI results will show the URL instead.

Reply:
Yeah, there are a few entries that are missing descriptions, but the spreadsheet attached to RHINENG-2398 doesn't have one for every entry. yet. 

I went over the apps under the care of the interact-team to confirm that those at least all have a description.

Reply:
Got it. It should be filled at some point. I'll merge the PR  and deploy the changes later today.